Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift

```diff
@@ -52,23 +58,23 @@ class CalibrationView: UIView {
     }()
     
     /// The last elevation slider value.
-    var lastElevationValue: Float = 0
+    private var lastElevationValue: Float = 0
     
     // The last heading slider value.
-    var lastHeadingValue: Float = 0
+    private var lastHeadingValue: Float = 0
```

Comment:
Does the comment above this need another slash?

Reply:
This property and the next were obsolete and not used, so I removed them.